src/features/Endpoint/Endpoint.module.scss

```diff
@@ -0,0 +1,104 @@
+.pageContent {
+  height: 140vh;
```

Comment:
can we remove this? 
its making the page big and adds a scrollbar, do we need it?